models/kubernetes_helpers_test.go

```diff
@@ -53,6 +56,18 @@ func TestKubernetesHelpers(t *testing.T) {
 				ID:                    "KC1",
 				KubernetesClusterName: "Kubernetes Cluster 1",
 				KubeConfig:            `{"kind": "Config", "apiVersion": "v1"}`,
+				PXC: &models.Component{
+					DisabledVersions: []string{"8.0.0"},
+					DefaultVersion:   "8.0.1-20",
+				},
+				ProxySQL: &models.Component{
+					DisabledVersions: []string{"8.0.0"},
+					DefaultVersion:   "8.0.1-19",
+				},
+				Mongod: &models.Component{
+					DisabledVersions: []string{"3.4.0", "3.6.0"},
```

Comment:
I am not sure these values have any effect. @BupycHuk

Reply:
Aside from that, it looks good to me.

Reply:
In this test we check only model layer, so values don't matter.